rust/wallet_core_rs/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "wallet-core-rs"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+name = "wallet_core_rs"
+crate-type = ["staticlib"] # Creates static lib
+
+[dependencies]
+tw_starknet = { path = "../tw_starknet" }
```

Comment:
Why is this dependencies needed, or why is it the only one?

Reply:
I have also added other dependencies. The main goal is to include extern FFI functions from `tw_starknet`, `tw_hash`, ... subcrates into the `wallet_core_rs` static library.